kuryr_kubernetes/handlers/retry.py

```diff
@@ -83,6 +83,11 @@ def __call__(self, event, *args, **kwargs):
                 }
                 self._handler(event, *args, retry_info=info, **kwargs)
                 break
+            except (exceptions.LoadBalancerNotReady,
+                    exceptions.PortNotReady):
+                with excutils.save_and_reraise_exception() as ex:
+                    if self._sleep(deadline, attempt, ex.value):
+                        ex.reraise = False
```

Comment:
Don't change it, it'll be less readable, but I just realized this could be `ex.reraise = not self._sleep(deadline, attempt, ex.value)`. :smiley: